lib/open-amp/Kconfig

```diff
@@ -18,3 +18,11 @@ config OPENAMP_RSC_TABLE_NUM_RPMSG_BUFF
 	help
 	  This option specifies the number of buffer used in a Vring for
 	  interprocessor communication
+
+config OPENAMP_RSC_TABLE_VRING_ALIGNMENT
+	int "Resource table Vring alignment"
+	default 16
+	depends on OPENAMP_RSC_TABLE
+	help
+	  This option specifies the alignment used in a Vring for 
+	  interprocessor communication
```

Comment:
I guess adding this option doesn't harm, but I didn't need to change the default value.

Reply:
you are right. We could see that it also work with 16. We reverted it